type-definitions/ts-tests/index.d.ts

```diff
@@ -1 +0,0 @@
-// Minimum TypeScript Version: 4.5
```

Comment:
Just tried out: tests are also passing on TS 4.4, but not on TS 4.3.

DefinitelyTyped is only testing versions of TypeScript that are less than 2 years old. Today this is TypeScript 4.7 and up.